src/network/network.cpp

```diff
@@ -132,7 +134,8 @@ uint32 NetworkPacket::GetCommand()
 {
 	if (data->size() >= sizeof(uint32)) {
 		return ByteSwapBE(*(uint32*)(&(*data)[0]));
-	} else {
+	}
+	else {
```

Comment:
Is there any reason for changing the style? The existing one matches the coding style that IntelOrca produced.